firebase-storage/firebase-storage.gradle

```diff
@@ -102,7 +102,7 @@ dependencies {
     testImplementation "com.google.truth:truth:$googleTruthVersion"
     testImplementation 'androidx.test:rules:1.2.0'
     testImplementation 'androidx.test:core:1.2.0'
-    testImplementation project(':appcheck:firebase-appcheck')
```

Comment:
afaik this is different (appcheck vs appcheck-interop), could you check if that was modified recently?